packages/components-core/src/components/RankingCard/index.js

```diff
@@ -1,56 +1,44 @@
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import React, { useState } from 'react';
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
 
-import { trackGAEvent } from '../../core/services/googleAnalytics';
-import { sanitizeDangerousMarkup } from '../../core/utils/html-utils';
+import { trackGAEvent } from "../../core/services/googleAnalytics";
+import { sanitizeDangerousMarkup } from "../../core/utils/html-utils";
+// eslint-disable-next-line import/no-cycle
+import { Image } from "../Image";
 
 const gaDefaultObject = {
-  name: 'onclick',
-  event: 'link',
-  action: 'click',
-  type: 'internal link',
-  region: 'main content',
+  name: "onclick",
+  event: "link",
+  action: "click",
+  type: "internal link",
+  region: "main content",
 };
 
 const AVAILABLE_GA_ACTIONS = {
-  OPEN: 'open',
-  CLOSE: 'close',
+  OPEN: "open",
+  CLOSE: "close",
 };
 
 const AVAILABLE_SIZES = {
-  LARGE: 'large',
-  SMALL: 'small',
+  LARGE: "large",
+  SMALL: "small",
 };
 
 const isSmallSize = size => size === AVAILABLE_SIZES.SMALL;
 
 const ImageWrapper = ({ size, image, imageAlt }) => {
   return isSmallSize(size) ? (
     <div className="image-wrapper">
-      <img
-        src={image}
-        alt={imageAlt}
-        loading="lazy"
-        decoding="async"
-        // eslint-disable-next-line react/no-unknown-property
-        fetchpriority="high"
-      />
+      <Image url={image} alt={imageAlt} fetchPriority="high" />
```

Comment:
Image doesn't render on Ranking Cards. I believe it's because `url` here should actually be `src`.